journal/api/src/controllers/locations.ts

```diff
@@ -0,0 +1,98 @@
+import { Elysia, t } from 'elysia';
+import {db} from "../db"
+import { eq } from 'drizzle-orm';
+import { createLocationsSchema, locations, locationsSchema } from '../db/schema';
+
+const identityParams = t.Object({
+  id: t.Numeric()
+})
+
+export const locationsController = new Elysia()
+    .decorate('db', db)
+
+///
```

Comment:
[NIT]: It would be nice if the method calls were on the same indent to keep the call chain easy to see.